packages/components/src/Banner/Banner.tsx

```diff
@@ -104,6 +104,7 @@ export function Banner({
 
       {dismissible && (
         <button
+          type="button"
```

Comment:
😅

Reply:
yeah, we have a linter now that reminds us to do that in case it goes in a form haha